test_task_packed_picking/tests/test_create_picking_and_put_product_a_package.py

```diff
@@ -0,0 +1,137 @@
+from odoo.tests import common, tagged
+
+
+@tagged("nice")
+class TestCreatePickingAndPutProductPackage(common.TransactionCase):
+    def setUp(self):
```

Comment:
Prefer setUpClass because it's faster and more relevant. 
Check comment [here](https://github.com/OCA/timesheet/pull/570#discussion_r1363198708)